src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CSharp.CodeFixes/HttpContextCurrentCodeFixer.cs

```diff
@@ -173,7 +173,7 @@ private static void InitializeHttpContextHelperInStartup(DocumentEditor editor,
             }
 
             // Add Initialize call in Configure method
-            var configureMethod = documentRoot.GetMethodDeclaration("Configure", "IApplicationBuilder");
+            var configureMethod = documentRoot.GetMethodDeclaration<MethodDeclarationSyntax>("Configure", "IApplicationBuilder");
```

Comment:
again, doesn't seem like you care about the CSharp syntax type. you just care that you found the right node.

Reply:
I do care because I use `configureMethod` as a MethodDeclarationSyntax object which is C#-specific.

Reply:
missed that somehow